PsfShimMonitor/App.xaml

```diff
@@ -0,0 +1,9 @@
+﻿<Application x:Class="TraceShimMonitor.App"
```

Comment:
should rename to PsfShimMonitor for consistency

Reply:
I'll be renaming the internal classes and references to PsfMonitor to match the change made to the exe name.

Reply:
Addressed in commit 727a9e9856d361f686523f2ea2907876c3ef9b23 on 11/28